common/src/main/java/com/wynntils/core/features/overlays/annotations/OverlayGroup.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.overlays.annotations;
+
+import com.wynntils.core.features.overlays.RenderState;
+import com.wynntils.mc.event.RenderEvent;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface OverlayGroup {
+    // How many instances of this overlay should exist by default
+    int instances() default 1;
```

Comment:
In fact, what would happen if you were to merge OverlayGroup and OverlayInfo? Moving the `instances()` method into OverlayInfo, and perhaps add a `boolean isGroup() default no`?

Reply:
Well, we would have to change reflections, and save having a class. I think I am going to refactor this either way, so I can merge this now, and remove it later, or just keep it as is, and remove it later.

Reply:
Ok, let's keep it this way and refactor later.